salesforce-composite/2.7/modules/ROOT/pages/salesforce-composite-connector-examples.adoc

```diff
@@ -245,14 +246,29 @@ payload]]></ee:set-payload>
 
 == Pre-Query Example
 
-This example provides metadata for an object to query, generates two subrequests that contain this metadata, and then executes the subrequests. 
-
-The following screenshot shows the flow for the Pre-Query example:
+This example provides metadata for an object to query, generates two subrequests that contain this metadata, and then executes the subrequests:
 
+* HTTP *Listener* +
```

Comment:
Kim dictates this style for lists:
HTTP Listener
+
Accepts data from HTTP requests

So basically, put the + on the next line, rather than at the end of the line.